Makefile.am

```diff
@@ -175,7 +175,7 @@ BATS = \
 	test/functional/bundleadd/add-experimental.bats \
 	test/functional/bundleadd/add-fall-back-to-fullfile.bats \
 	test/functional/bundleadd/add-include.bats \
-	test/functional/bundleadd/add-include-optional.bats \
+	test/functional/bundleadd/add-also-add-flag.bats \
```

Comment:
since the file changed names you need to reorder the files, remember they are sorted alphabetically